socketsClient.js

```diff
@@ -1,49 +1,49 @@
 var spawn = require('child_process').spawn;
 var net = require('net');
-var cfv = require('./confVars.js');
+var cfv = require('./confVars');
 
 //var result = [];
-var instances = cfv.sensorArr.length;
 
 
 function getStationsData(callback) {
-	var result = [];
-	if (cfv.sensorArr.length == cfv.addrArr.length && cfv.portArr.length == cfv.sensorArr.length){
-		for (var i = 0; i<cfv.sensorArr.length; i++){
-			fireSocket(cfv.addrArr[i],cfv.portArr[i],cfv.sensorArr[i],callback,result);
-		}
-	} else {
-		console.log("sensorArr: ",sensorArr.length ,"\naddrArr: ",  addrArr.length ,
-		"\nportArr: ", portArr.length);
-		console.log("Those arrays should be of equal length");
-		throw new Error("ParametersError: Data Array Length not Consistent");
-	}
+    var instances = cfv.sensorArr.length;
```

Comment:
this variable should be initialized only on module load because it will not change along the process and is shared between all sockets.